src/actions.rs

```diff
@@ -134,7 +135,7 @@ enum Instruction {
     CallPrimitive(Primitive, usize),
     /// Pop function arguments off the stack and delete the corresponding row
```

Comment:
This documentation is not correct.
Could you please change it to give an example of `delete`, and example of using `subsume`, and what each of them means?

Reply:
I fixed this line. The docs for `Change` should cover what each arm does in more detail, so it looks sufficient now to me compared to the docs for the other Instructions.